mat64/dense.go

```diff
@@ -379,7 +379,9 @@ func (m *Dense) Clone(a Matrix) {
 
 // Copy makes a copy of elements of a into the receiver. It is similar to the
 // built-in copy; it copies as much as the overlap between the two matrices and
-// returns the number of rows and columns it copied.
+// returns the number of rows and columns it copied. If a aliases the receiver
+// and is a transposed Dense or Vector, with a non-unitary increment, Copy will
+// panic.
```

Comment:
I don't see the need to need for this proliferation. Copy does the work properly. An alternative approach which I also dislike, but less, is to rename Vector to Column and make another type Row that is defined `type Row Column` but satisfies Matrix.

The view approach you have above is fine for copying a row from one matrix to another, but doesn't deal with the case where you are building rows of a matrix from operations that produce a vector.